docs/ops/detection/ROIPooling_1.md

```diff
@@ -44,13 +44,17 @@
 
 **Inputs**:
 
-*   **1**: 4D input tensor of shape `[1, C, H, W]` with feature maps. Required.
+*   **1**: 4D input tensor of shape `[1, C, H, W]` with feature maps of type *T*. Required.
```

Comment:
The batch size of this input tensor may be arbitrary number `N` corresponding to number of input images.

The second tensor input with tuples `[batch_id, x_1, y_1, x_2, y_2]` values for elements `batch_id` must be in the range `[0, batch_id - 1]`